vignettes/ectd/tlf-efficacy.R

```diff
@@ -147,8 +146,15 @@ tbl_1 <- apr0ancova1 %>%
       "CI = Confidence Interval, LS = Least Squares, SD = Standard Deviation"
     )
   ) %>%
+  rtf_footnote(
+    footnote = c(
+      paste("Table generated on:", Sys.time())
+    ),
+    as_table = FALSE,
+    text_justification = "c"
+  ) %>%
```

Comment:
@kaz462 feel free to use this code to add the run date footnote to your table.

Reply:
Thanks, @DeclanHodges !

Reply:
@laxamanaj I just realized the timestamp was already available for `tlf-primary.r`, so no further updates are needed

Reply:
Thanks, @DeclanHodges and @kaz462 !  Good to know.  Update looks great!